ui/app/mainui/Popups.qml

```diff
@@ -467,6 +479,23 @@ QtObject {
             }
         },
 
+        Component {
+            id: reviewContactRequestPopupComponent
+            ReviewContactRequestPopup {
+                onAccepted: {
+                    rootStore.contactStore.acceptContactRequest(publicKey, contactRequestId)
+                    Global.displaySuccessToastMessage(qsTr("Contact request accepted"))
+                    close()
+                }
+                onDiscarded: {
+                    // FIXME should call `rootStore.contactStore.dismissContactRequest(publicKey, contactRequestId)` here?
+                    Global.displaySuccessToastMessage(qsTr("Contact request ignored"))
```

Comment:
One question here @MishkaRogachev: here we should just "ignore" the request but all the other popups (ActivityCenter, Settings/Contacts, etc) still call the `dismissContactRequest`; so what to do here?

Reply:
i'm not aware of latest requirements, but i can answer what `dismissContactRequest` does:
- mark provided or latest CR as _Dismissed_
- set contact's `ContactRequestLocalState` as ContactRequestStateDismissed. All subsequent CRs from this contact will not change the contact's status and trigger AC notifications

So my opinion is to use `dismissContactRequest` to _ignore_ a contact

Reply:
Thanks, done